Packs/Base/Scripts/DBotMLFetchData/DBotMLFetchData.yml

```diff
@@ -45,7 +50,8 @@ args:
 - auto: PREDEFINED
   default: false
   defaultValue: created
-  description: 'The incident field to specify for the date range. Can be: "created" or "modified". The default is "created".'
+  description: 'The incident field to specify for the date range. Can be: "created"
```

Comment:
the default time query should be modified, no?

Reply:
this argument is not actually in use, I don't think I can delete it because of backwards compatibility